java/src/jmri/jmrit/logix/PortalManager.java

```diff
@@ -45,7 +45,7 @@
     private PropertyChangeSupport pcs = new PropertyChangeSupport(this);
     private ArrayList<Portal> _nameList = new ArrayList<>();   // stores Portal in loaded order
     private Hashtable<String, Portal> _portalMap = new Hashtable<>();   // stores portal by current name
-    private Integer _nextIndex = new Integer(1);
+    private Integer _nextIndex = Integer.valueOf(1);
```

Comment:
You can write this line as:
`private Integer _nextIndex = 1;`
Autoboxing in Java will convert the number to an Integer.